src/jwk.rs

```diff
@@ -388,6 +388,32 @@ where
             if let Err(e) = policy.algorithm(alg) {
                 return Err((self, e));
             }
+
+            let operations = match alg {
+                JsonWebAlgorithm::Encryption(..) => &[
```

Comment:
Just use an array instead of a slice. `[Operation::Encrypt, Operation::Decrypt]` and `[Operation::Sign, Operation::Verify]`